src/app/donation-complete/donation-complete.component.html

```diff
@@ -14,115 +17,97 @@
+      </p>
+
+      <p *ngIf="registerError" class="error" aria-live="assertive">
+        Your password could not be set. Please ensure you chose one with at least 10 characters.
+      </p>
+    </div>
+
+    <hr>
+
+    <div class="donation-summary">
+      <p><strong>Hi Ali,<br/>Your donation summary</strong></p>
+      <p>Your donation of <strong class="figure">{{ donation.donationAmount | exactCurrency:donation.currencyCode }}</strong> has been made. That's worth <strong class="figure">{{ totalValue | exactCurrency:donation.currencyCode }}</strong> to {{ donation.charityName }}!</p>
+
+      <br/>
+      <p *ngIf="campaign?.thankYouMessage">{{ campaign.thankYouMessage }}</p>
+    </div>
+
+    <hr>
+
+    <div class="receipt">
+      <p><strong>Your Receipt</strong></p>
+      <p>{{donation.createdTime | date: 'medium'}}</p>
+      <br>
+
+      <table>
+        <tr>
+          <td>Your donation</td>
+          <td>{{ donation.donationAmount | exactCurrency:donation.currencyCode }}</td>
+        </tr>
+        <tr *ngIf="donation.matchedAmount > 0">
+          <td>Matched Amount</td>
+          <td>{{ donation.matchedAmount | exactCurrency:donation.currencyCode }}</td>
+        </tr>
+        <tr *ngIf="giftAidAmount > 0">
+          <td>Gift Aid Amount</td>
+          <td>{{ giftAidAmount | exactCurrency:donation.currencyCode }}</td>
+        </tr>
+        <tr *ngIf="donation.tipAmount > 0">
+          <td>Tip Amount</td>
+          <td>{{ donation.tipAmount | exactCurrency:donation.currencyCode }}</td>
+        </tr>
+        <tr class="receives">
+          <td>Charity Receives</td>
+          <td>{{ totalValue | exactCurrency:donation.currencyCode }}</td>
+        </tr>
+        <tr class="total">
+          <td><strong>Total amount you have been charged</strong></td>
+          <td><strong>{{ cardChargedAmount | exactCurrency:donation.currencyCode }}</strong></td>
+        </tr>
+      </table>
+    </div>
+
+    <hr>
+
+    <div class="socials">
+      <p>Boost your impact by sharing on social media</p>
+      <div class="social-icons-wrapper">
+        <biggive-social-icon service="Facebook" url="https://www.facebook.com/sharer/sharer.php?u={{ shareUrl }}" background-colour="tertiary" icon-colour="black" wide="true"></biggive-social-icon>
+        <biggive-social-icon service="Twitter" url="https://twitter.com/share?url={{ shareUrl }}&text={{ prefilledText }}" background-colour="tertiary" icon-colour="black" wide="true"></biggive-social-icon>
+        <biggive-social-icon service="LinkedIn" url="https://www.linkedin.com/cws/share?url={{ shareUrl }}" background-colour="tertiary" icon-colour="black" wide="true"></biggive-social-icon>
+        <!-- <biggive-social-icon service="Whatsapp" url="https://api.whatsapp.com/send?text={{ prefilledText }}%0A%0A{{ shareUrl }}" background-colour="tertiary" icon-colour="black" wide="true"></biggive-social-icon> -->
```

Comment:
We need a WhatsApp icon baked into the Components library in order to add the WhatsApp share icon here.

Reply:
Never mind, I've added the icon in this Components PR: https://github.com/thebiggive/components/pull/117, and I'll update this PR accordingly to use it..

Reply:
Fixed now @bdsl